py/server/deephaven/parquet.py

```diff
@@ -6,14 +6,14 @@
 Parquet files. """
 from dataclasses import dataclass
 from enum import Enum
-from typing import List, Optional, Union, Dict
+from typing import List, Optional, Union, Dict, Literal
```

Comment:
```suggestion
from typing import List, Optional, Union, Dict
```
If we aren't using `Literal`, this doesn't belong.